tools/clang/include/clang/SPIRV/ModuleBuilder.h

```diff
@@ -150,23 +150,39 @@ class ModuleBuilder {
                     uint32_t defaultLabel,
                     llvm::ArrayRef<std::pair<uint32_t, uint32_t>> target);
 
-  // \brief Creates an unconditional branch to the given target label.
+  /// \brief Creates an unconditional branch to the given target label.
   void createBranch(uint32_t targetLabel);
 
-  // \brief Creates a conditional branch. An OpSelectionMerge instruction
-  // will be created if mergeLabel is not 0 and continueLabel is 0.
-  // An OpLoopMerge instruction will also be created if both continueLabel
-  // and mergeLabel are not 0. For other cases, mergeLabel and continueLabel
-  // will be ignored. If selection control mask and/or loop control mask are
-  // provided, they will be applied to the corresponding SPIR-V instruction.
-  // Otherwise, MaskNone will be used.
+  /// \brief Creates an OpLoopMerge instruction followed by an uncondtiional
```

Comment:
I think we can merge this overload with the above by giving `mergeBB` and `continueBB` defaults (0)?